collectors/pod.go

```diff
@@ -137,6 +137,18 @@ var (
 		"The limit on memory to be used by a container in bytes.",
 		[]string{"namespace", "pod", "container", "node"}, nil,
 	)
+
+	descPodContainerResourceRequestsGPU = prometheus.NewDesc(
+		"kube_pod_container_resource_requests_gpu",
```

Comment:
@WIZARD-CXY Thanks for providing correct info!

> it actually denotes one gpu device afaik.

Can this be renamed to `kube_pod_container_resource_limits_gpu_devices`? This is only my thought and i am opening to other thoughts. The intention behind this is that we need to make the unit for a metric clear. :)